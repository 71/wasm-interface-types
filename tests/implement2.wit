;; parse-fail

(module
  (import "" "" (global i32))
  (@interface implement (import "" "") (func 1)))
