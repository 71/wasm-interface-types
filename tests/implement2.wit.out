import of `` from `` not found in core module
     --> ../../tests/implement2.wit:5:15
      |
    5 |   (@interface implement (import "" "") (func 1)))
      |               ^
